File: Back-end/src/main/java/com/example/demo/entity/ClientAccept.java
package com.example.demo.entity;

import java.sql.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.OneToOne;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;


@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class ClientAccept {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	private String acceptation;
	private String name;
	private String mail;
	private String message;
	private Date addedDate;
    @OneToOne
    @JoinColumn(name = "cautch_id")
    private User user;
}

File: src/app/clientaccept.ts
export class clientaccept {
    id: number;
    clientaccept:String;
    name:string;
    addedDate:Date;
    mail:string;
    acceptation:string;
    message:string;
   constructor(data: any = {}) {
     this.id= data.id|| 0;
     this.clientaccept = data.clientaccept || '';
     this.addedDate = data.addedDate;
     this.name=data.name || '';
     this.mail=data.mail || '';
     this.acceptation=data.acceptation || '';
     this.message=data.message || '';
   }
 }